src/web/releases.rs

```diff
@@ -834,10 +843,14 @@ pub(crate) async fn build_queue_handler(
             })
         })
         .map(|mut krate| {
```

Comment:
I think this should be `retain_mut` as in [my WIP branch](https://github.com/rust-lang/docs.rs/compare/master...syphar:docs.rs:group-build-queue?expand=1#diff-c3796aac6c6b01cd1998b29108b5d32fe5113970a7f4558e799c18984307d73aR819), or am I missing something?